examples/Makefile

```diff
@@ -35,9 +35,9 @@
 
 .DEFAULT_GOAL := transform
 
-.PHONY: all transform compile clean allclean
+.PHONY: all transform compile notebook clean allclean
 
-all transform compile clean allclean:
+all transform compile clean allclean notebook:
```

Comment:
Can you please place notebook before clean in the list.

Reply:
Done.